testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketConnectTest.java

```diff
@@ -82,7 +84,7 @@ public void testChannelEventsFiredWhenClosedDirectly(ServerBootstrap sb, Bootstr
         Channel cc = null;
         try {
             sb.childHandler(new ChannelInboundHandlerAdapter());
-            sc = sb.bind(0).syncUninterruptibly().channel();
+            sc = sb.bind(NetUtil.LOCALHOST, TestUtils.getFreePort()).syncUninterruptibly().channel();
```

Comment:
IIRC `TestUtils.getFreePort()` has been observed to fail when test are run in quick succession due to bind failures (address already in use). Can we just use `new InetSocketAddress(0)` which will pick an ephemeral port instead?